Add new Java Math Methods entry: `.sqrt()` method (#936)
Codecademy/docs: content/java/concepts/math-methods/terms/sqrt/sqrt.md

---
Title: '.sqrt()'

> Description: 'Returns the square root of a value of type double passed to it as argument.'

Reviewer: ```suggestion
Description: 'Returns the square root of a double value passed to it as argument.'
```

Subjects:
  - 'Computer Science'
Tags:

>   - 'Methods'
>   - 'Arithmetic'
>   - 'Sqrt'

Reviewer 2: ```suggestion
  - 'Arithmetic'
  - 'Functions'
  - 'Methods'
  - 'Numbers'
```

For the meta-`Tags`, let's use what's inside the [tags.md](https://github.com/Codecademy/docs/blob/main/documentation/tags.md) file. I suggest adding `'Functions'` for SEO-purposes, as well as the `'Numbers'` tag.

CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`Math.sqrt()`** method returns the ` Square root ` of a number.

Reviewer: ```suggestion
The **`Math.sqrt()`** method returns the square root of a double value passed to it as argument.
```

> The **`Math.sqrt()`** method returns the square root of a double value passed to it as an argument.

Reviewer 2: ```suggestion
The **`Math.sqrt()`** method returns the positive, properly rounded square root of a double-type value.
```

## Syntax

```pseudo
Math.sqrt(double num)
```

> The `.sqrt()` method takes one parameters.

Reviewer: ```suggestion
The `.sqrt()` method takes one parameter.
```

> The `.sqrt()` method takes one parameter.

Reviewer 2: ```suggestion
The return type of the `.sqrt()` method is a double. The data type of the `num` parameter is also a double. However, if a decimal is not provided, `num` will still be treated like a double-type.

Some edge cases for the `.sqrt()` method include the following:

 - If `num` is less than zero or it is not a numeric type (`NaN`), then `NaN` is returned.
 - If `num` is positive infinity (`Double.POSITIVE_INFINITY`), then `Infinity` is returned. 
```

The "Syntax" section is a good place to list out, if any, the edge cases of parameters used in a method/function.

## Example

> To show working of `Math.sqrt()` method.  

Reviewer: ```suggestion
This example shows the working of the `Math.sqrt()` method.  
```

> This example shows the working of the `Math.sqrt()` method.  

Reviewer 2: ```suggestion
The following example showcases the `.sqrt()` method being applied to two variables:
```

```java
public class Check {
  public static void main(String args[]) {
       double x = 64.0;  
       System.out.println(Math.sqrt(x));
       double y = 49.0;
       System.out.println(Math.sqrt(y));
   }
}
```

> ## Output:

Reviewer: ```suggestion
This results in the following output:
```
